### PR TITLE
Update TreeSetCharacters.java

### PR DESCRIPTION
**Pull request by joej0e**

### CODE REVIEW

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 1]
```diff
@@ -17,7 +23,22 @@
...
+        try (BufferedReader br = new BufferedReader(new FileReader(fileName))) {
+            Set<Character> set = new TreeSet<>();
+            int c = br.read();
+            while (c != -1) {
+                if (Character.isAlphabetic((char) c)) {
+                    set.add((char) Character.toLowerCase(c));
+                }
+                c = br.read();
+            }
+            if (set.size() > 5) {
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]: @joej0e what means 5?

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/TreeSetCharacters.java (hunk 1 above)
Comment (boroda4436) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: move to constant

File: src/main/java/core/basesyntax/TreeSetCharacters.java [hunk 2]
```diff
@@ -17,7 +23,25 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+
+    private final static int maxSise = 5;
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/TreeSetCharacters.java]: @joej0e Constants should be named WITH_CAPSLOCK_AND_UNDERSCRORE

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/TreeSetCharacters.java (hunk 2 above)
Comment (joej0e) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: Travis said me that i can't 
Abbreviation in name 'MAX_SIZE' must contain no more than '2' consecutive capital letters. [AbbreviationAsWordInName]
[ERROR] /home/travis/build/mate-academy/jv-tree-set/src/main/java/core/basesyntax/TreeSetCharacters.java:27:23: Member name 'MAX_SIZE' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'. [MemberName]

Comment (boroda4436) [reply src/main/java/core/basesyntax/TreeSetCharacters.java]: OK

**Pull request closed without merging**